android/src/main/java/com/tailscale/ipn/ui/viewModel/MainViewModel.kt

```diff
@@ -132,10 +143,6 @@ class MainViewModel(private val vpnViewModel: VpnViewModel) : IpnViewModel() {
         }
       }
 
-      viewModelScope.launch {
-        searchTerm.collect { term -> peers.set(peerCategorizer.groupedAndFilteredPeers(term)) }
-      }
-
       viewModelScope.launch {
```

Comment:
I believe the bug was introduced when a } was deleted and this block and the above block launching the searchTerm.collect became unreachable. Could you also move the `viewModelScope.launch {
      App.get().healthNotifier?.currentIcon?.collect { icon -> healthIcon.set(icon) }
    }` as well please?

Reply:
I've made those changes @kari-ts